samples/zigbee/template/src/main.c

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ *
+ * @brief Zigbee application template.
+ */
+
+#include <zephyr.h>
+#include <logging/log.h>
+#include <dk_buttons_and_leds.h>
+
+#include <zboss_api.h>
+#include <zigbee/zigbee_error_handler.h>
+#include <zigbee/zigbee_app_utils.h>
+#include <zb_nrf_platform.h>
+
+
+/* Device endpoint, used to receive ZCL commands. */
+#define APP_TEMPLATE_ENDPOINT           10
+
+/* Type of power sources available for the device.
+ * For possible values see section 3.2.2.2.8 of ZCL specification.
+ */
+#define BULB_INIT_BASIC_POWER_SOURCE    ZB_ZCL_BASIC_POWER_SOURCE_DC_SOURCE
+
+/* LED indicating that device successfully joined Zigbee network. */
+#define ZIGBEE_NETWORK_STATE_LED        DK_LED3
+
+/* LED used for device identification. */
+#define IDENTIFY_LED                    DK_LED4
+
+/* Button used to enter the Identify mode. */
+#define IDENTIFY_MODE_BUTTON            DK_BTN4_MSK
+
+
+LOG_MODULE_REGISTER(app);
+
+/* Main application customizable context.
+ * Stores all settings and static values.
+ */
+struct zb_device_ctx_s {
```

Comment:
nit: We don't typically add `_s` suffix in NCS codebase (I guess it still falls under "hungarian notation")?

Reply:
Removed :slightly_smiling_face: